lib/log_ignored_message/log_ignored.rb

```diff
@@ -0,0 +1,62 @@
+module LogIgnoredMessage
+  class LogIgnored
```

Comment:
maybe we can add it to eventide namespace:

```suggestion
class Log
  class IgnoredMessage
```
but you need to require to load Log namespace first and dependency could be: `:log_ignored_message`

Reply:
and to be honest I think it should still be a logger dependency, which has method `ignore_message`

because we could need `logger.info` in some components. We might need to log some other info than ignored message, so it will require to include second logger into class, what could be weird.

It dependency could inherit from Log or delegate at some of methods, like:
:fatal, :error, :warn, :info, :debug, :trace, which are default log levels

Reply:
Oh!  we even could create our own log level: `ignore_message` which could have his own formatter. his level could be equal to info level.

And if we will change the way we include logger in our class it should be possible to use eventide substitute without tag: :ignore, which might allow test it without extra effort/configuration in tests

It will require some investigation, but I can do it if you are interested

Reply:
I've kept it simple for now, and simply removed the namespace and made it `dependency LogIgnoredMessage, :log_ignore_message`